OpenUpSA/wazimap-ng-ui
# updating key metrics version indicator when user switches to another version

## Description
* key metrics are always shown even when the key metric is not from the selected geo version
* if the key metric is not from the selected version, there is a notification below the key metric indicating which geo version it is from

## Related Issue
https://wazimap.atlassian.net/browse/WNCM-139

## How is it tested automatically?
added test steps to `__tests__/gui/multi_boundary_versions.feature`

## How should a reviewer test it locally
* go to SANEF profile 
* set the API url staging
* confirm that key metric is shown and if it is not from the selected geo version, there is a note below it
* confirm that this works everytime by reloading the page or
        - navigating to other geographies 
        - changing the selected geo version
        - navigating back to ZA


## Screenshots
* key metric is from the selected geo version
![image](https://user-images.githubusercontent.com/53019884/141021686-e20fef12-2fe7-4c92-832c-0953d40ca661.png)

* key metric is not from the selected geo version
![image](https://user-images.githubusercontent.com/53019884/141021781-949b3c29-d5ea-4bc1-ac9b-48006ef28b6d.png)


## Changelog

### Added
* removed the key metric related code from `src/js/profile/subcategory.js` and created `src/js/profile/key_metric.js` 

### Updated
* `src/js/versions/version_controller.js` to make sure all the key metrics from all the available geo versions are shown

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
